### ISSUE TITLE
Splitting off matches into their own file?

### ISSUE THREAD

**Issue by mpentler** [bartnv/portlurker#7]
I've been wondering if it's possible to improve this at all: https://github.com/bartnv/portlurker/blob/6a629fdf0f5126ef8d570a0a68d5b61ca6338b20/src/main.rs#L18

I hate that we have to keep track of how much is in there for the global array. I think the only reason I did it was to put it into another file, so they need to be globals for it to work.

Instead we could maybe put the matches into a separate file in another format and parse it, and count the lines to see how many there are when defining the array. Easier to keep track of then.

What do you think?

**Comment by bartnv** [comment]
Yeah, having those binary matches defined in a separate configuration file would certainly be preferable. They're tuples of a normal string and a raw string. Not sure how you can read text from a file and have it interpreted as a raw string (with the \x replacements executed). That's the only hard part. The two strings could just be tab-separated on the same line.

**Comment by mpentler** [comment]
What, have them separated so you could read one part as a normal and one as a raw?

Or you could do it as YAML? Easier to parse then.
